pyproject.toml

```
@@ -14,7 +14,8 @@ classifiers = [
 ]
 dependencies = [
     "astropy >=5.0.4",
-    "drizzle>=1.15.0",
+    # "drizzle>=2.0.0",
+    "drizzle @ git+https://github.com/mcara/drizzle.git@disable-bbox",
```

Is there a PR open for this?

---

```suggestion
    "drizzle @ git+https://github.com/mcara/drizzle.git@main",
```
It looks like it was https://github.com/spacetelescope/drizzle/pull/164. My vote is to release this and add a lower pin for the released version.